geode-core/src/main/java/org/apache/geode/internal/cache/partitioned/QueryMessage.java

```diff
@@ -169,7 +169,9 @@ protected boolean operateOnPartitionedRegion(ClusterDistributionManager dm, Part
 
     DefaultQuery query = new DefaultQuery(this.queryString, pr.getCache(), false);
     // Remote query, use the PDX types in serialized form.
-    DefaultQuery.setPdxReadSerialized(pr.getCache(), true);
+    Boolean initialPdxReadSerialized =
```

Comment:
Same question, any chance the registry is null for these calls?

Reply:
there seems to be a possibility that the registry can be null. Will add the check.